cpp/include/cudf/functions.h

```diff
@@ -381,19 +381,23 @@ gdf_error gdf_hash_partition(int num_input_cols,
 /* prefixsum */
 
 /* --------------------------------------------------------------------------*/
-/** 
- * @brief  Computes the prefix sum of a column
- * 
- * @param inp Input column for prefix sum with null_count = 0
- * @param out The output column containing the prefix sum of the input
- * @param inclusive Flag for applying an inclusive prefix sum
- * 
- * @returns   GDF_SUCCESS if the operation was successful, otherwise an appropriate
- * error code. If inp->null_count is not set to 0 GDF_VALIDITY_UNSUPPORTED is
- * returned.
+/**
+ * @brief  Computes the prefix sum of a column.
+ * The Null values are skipped for the operation,
+ * and the output column has same null values with the input column.
```

Comment:
```suggestion
 * and if an input element at `i` is null, then the output element at `i` will also be null. 
```

Reply:
fixed merge fault and updated doxygen text.